release_bot/configuration.py

```diff
@@ -20,6 +20,9 @@
 from pathlib import Path
 
 import yaml
+from ogr.services.github import GithubService
```

Comment:
This should work as well:

```suggestion
from ogr import GithubService
```